modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/fragments/group/commerce/fragments/header-main/index.html

```diff
@@ -0,0 +1,17 @@
+<div class="border-bottom py-2" id="commerce-header-main">
+	<div class="align-items-center container-fluid container-fluid-max-xl d-flex justify-content-between">
+		<div id="commerce-header-logo">
+			<img alt="" data-lfr-editable-id="commerce-site-footer-logo" data-lfr-editable-type="image" src="[resources:site_logo.svg]" />
+		</div>
+
+		<div id="commerce-header-search">
+			[@liferay_commerce_ui["search-bar"] /]
```

Comment:
@gianmarcobrunialti can we use the OOB search bar providing a template?

Reply:
Hey @marco-leo, were handling this in a separate ticket, next to this, can I put this up to the other PR or would you prefer to include this here immediately?

Reply:
Sure, good to hear